_content/essays/buddhism-and-semiotics_fabio-rambelli.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Buddhism and Semiotics"
+authors:
+  - "Fabio Rambelli"
+external_url: "https://www.thezensite.com/ZenEssays/Philosophical/Buddhism_and_Semiotics.html"
+drive_links:
+  - "https://drive.google.com/file/d/1FMTye0ePjhE3KI1cAdxp9wYdOmUHtTWA/view?usp=sharing"
+course: dialogue
+tags:
+  - east-asian
+  - yogacara
+  - language
+publisher: "The Zen Site"
```

Comment:
Essays don't get / need a publisher but do need a year.

But in any case this was originally an _article_ (yes please move there) published in 1995 in The Semiotic Review of Books (`journal: `)